libraries/plugins/witness/witness_plugin.cpp

```diff
@@ -603,7 +605,7 @@ namespace detail {
          return block_production_condition::low_participation;
       }
 
-      if( llabs((scheduled_time - now).count()) > fc::milliseconds( 500 ).count() )
+      if( llabs((scheduled_time - now).count()) > fc::milliseconds( 750 ).count() )
```

Comment:
Change 750 to a defined constant at the top of the source file. Yes, this isn't what was written when you started your change, but it was done wrong previously.